### PR TITLE
virsh_snapshot_create_as: Check for the package "targetcli"

### PR DESCRIPTION
**Pull request by MalleshKoti**
…s installed and install if it is not there.

Got Error to install targetcli package to run the test so added code to install targetcli.

Signed-off-by: Mallesh Koti <mallesh@linux.vnet.ibm.com>

### CODE REVIEW

File: libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_create_as.py [hunk 1]
```diff
@@ -410,6 +411,9 @@ def run(test, params, env):
     vmxml_backup = vm_xml.VMXML.new_from_inactive_dumpxml(vm_name)
     logging.debug("original xml is %s", vmxml_backup)
 
+    if not utils_package.package_install('targetcli'):
```
Comment (balamuruhans) [inline libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_create_as.py]: This looks to be command, we should consider package name for installation.
`tgt` is the package name for Ubuntu based distro and `scsi-target-utils` for rhel based distro.

so IMO you can check whether this command is available or not and then proceed the package installation accordingly.

Comment (waynesun09) [reply libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_create_as.py]: NACK

please check virttest/iscsi.py module, both targetcli and tgtadm will be checked and raise error to user for install the package.

You could add the package in your env preparation as required package base on OS, as iscsi is widely used in this test suite.

Comment (balamuruhans) [reply libvirt/tests/src/virsh_cmd/snapshot/virsh_snapshot_create_as.py]: @MalleshKoti I have handled it from framework, https://github.com/avocado-framework/avocado-vt/pull/1161, please help to check whether it resolves your issue.

**Comment by iccaszhulili** [comment]
rh-caselink retest please

**Comment by dzhengfy** [comment]
After confirmation with the IBM PPC FVT lead that the author does not work on this any more and the pr is not in use by them. So I close it.

**Pull request closed without merging**